collectors/feature/jira/src/main/java/com/capitalone/dashboard/utils/Utilities.java

```diff
@@ -0,0 +1,31 @@
+package com.capitalone.dashboard.utils;
+
+import org.json.simple.JSONObject;
+
+public class Utilities {
+    public static String getString(JSONObject json, String key) {
+        if (json == null) return "";
+        Object o = json.get(key);
+        if (o == null) return "";
+        if (o instanceof Double) {
+            Double d = (Double) o;
+            return String.valueOf(d.intValue());
+        }
+        return String.valueOf(o);
+    }
+
+    public static long getLong(JSONObject json, String key) {
+        if (json == null) return 0;
+        Object o = json.get(key);
+        if (o == null) return 0;
+        return (Long) o;
+    }
+
+    //This is weird but way faster than Java date time formatter etc.
+    public static String parseDateWithoutFraction(String date) {
```

Comment:
@Sbrenthughes  what is date format in input args ?

Reply:
Unit tests added.